### DISCUSSION TITLE
Cable connection one to many?

### DISCUSSION THREAD

**Discussion by salvadorb** [netbox-community/netbox#15175] [General]
Hi Netbox community,
I hope you're doing well. In many cases we use MPO breakout cables to connect one device to multiple interfaces in other device, but in other cases we use MPO breakout to connect one device to interfaces in multiple devices (for example: 1 100GBASE-SR4 interface in the switch connected to via MPO breakout to four servers). Unfortunately I haven't find a way to use the interface connection to multiple devices, it only allows multiple interfaces in a single device. Is there any plan to include that? Or am I missing something? 

I'm currently using as a workaround to add multiple "virtual" interfaces (like eth1/1, eth1/2, eth1/3, eth1/4) instead of a single interface, but I would prefer the other way.

But as you can see, it doesn't look good
<img width="1405" alt="image" src="https://github.com/netbox-community/netbox/assets/23199382/908d393e-291a-450b-8984-46b90e160350">

Best regards,

**Comment by candlerb** [comment]
What's your Netbox version?

**Reply by salvadorb** [reply]
v.3.5.6. Was it already implemented after that version?

**Reply by candlerb** [reply]
No, it was v3.3.0 I believe.

I think you're probably just finding the UI confusing. When you add or edit a cable, you can select multiple terminations.

![image](https://github.com/netbox-community/netbox/assets/44789/5d46d073-8af0-41ed-a930-3127c88e2857)

But you are allowed to change the selected "Device", then click the "+" button next to Interface, and then you can select interfaces from that new device, and so on.

However, the list of interfaces shown only gives their names, not the devices they belong to. This can be especially confusing if you're connecting to the same named interface on multiple devices (which is quite common). #10841 shows an example of that.

(That screenshot from v3.7.2. There was a UI change which introduced the search button, and I'm not sure when that was, but even before then you could select the device from the dropdown)

**Comment by salvadorb** [comment]
v3.5.6

On Fri, Feb 16, 2024 at 2:06 PM Brian Candler ***@***.***>
wrote:

> What's your Netbox version?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/netbox-community/netbox/discussions/15175#discussioncomment-8496196>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFQ75FQ2NWX4OTZ5BJ3EFWLYT6U25AVCNFSM6AAAAABDMDUMVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJWGE4TM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Salvador Bertenbreiter
(+51) 947 145 352
(+57) 323 802 9009

**Comment by salvadorb** [comment]
Hi,
Yes, but all terminations (on one side) must end on the same device, that's
the limitation I'm facing. I would like to put 1 interface in side A, but
in side Z add interface X in device 1 and interface Y in device 2

On Fri, Feb 16, 2024 at 4:00 PM Brian Candler ***@***.***>
wrote:

> No, it was v3.3.0 I believe.
>
> I think you're probably just finding the UI confusing. When you add or
> edit a cable, you can select multiple terminations.
>
> image.png (view on web)
> <https://github.com/netbox-community/netbox/assets/44789/5d46d073-8af0-41ed-a930-3127c88e2857>
>
> But you are allowed to change the selected "Device", then click the "+"
> button next to Interface, and then you can select interfaces from that new
> device, and so on.
>
> However, the list of interfaces shown only gives their names, not the
> devices they belong to. This can be especially confusing if you're
> connecting to the same named interface on multiple devices (which is quite
> common). #10841 <https://github.com/netbox-community/netbox/issues/10841>
> shows an example of that.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/netbox-community/netbox/discussions/15175#discussioncomment-8497029>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFQ75FTBEFOZVT6WS53EGPTYT7CHRAVCNFSM6AAAAABDMDUMVWVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJXGAZDS>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 
Salvador Bertenbreiter
(+51) 947 145 352
(+57) 323 802 9009

**Reply by candlerb** [reply]
> Yes, but all terminations (on one side) must end on the same device

That's not true, at least, at least not for any version I've used - I don't have a v3.5.6 to test with. But it certainly works with v3.7.2.

At the Z side you simply select device 1, add interface X, select device 2, add interface Y.

**Comment by deanfourie1** [comment]
I assume this does not apply to wireless links, creating multiple connections for wireless links
